src/Raven.Server/Config/Categories/HttpConfiguration.cs

```diff
@@ -40,6 +40,18 @@ public HttpConfiguration()
         [ConfigurationEntry("Http.MaxRequestLineSizeInKb", ConfigurationEntryScope.ServerWideOnly)]
         public Size MaxRequestLineSize { get; set; }
 
+        [Description("Set Kestrel's HTTP2 keep alive ping timeout")]
+        [DefaultValue(null)]
+        [TimeUnit(TimeUnit.Seconds)]
+        [ConfigurationEntry("Http.KeepAlivePingTimeoutInSec", ConfigurationEntryScope.ServerWideOnly)]
```

Comment:
We should use?

```suggestion
        [ConfigurationEntry("Http2.KeepAlivePingTimeoutInSec", ConfigurationEntryScope.ServerWideOnly)]
```

Reply:
Went with 'Http.Http2.KeepAlivePingDelayInSec'
Http -> section
Http2 -> subsection

Like we have with Indexing.Corax.Sth or Indexing.Lucene.Sth